src/overlays/actors/ovl_En_Invadepoh/z_en_invadepoh.c

```diff
@@ -20,492 +251,4503 @@ const ActorInit En_Invadepoh_InitVars = {
+        } else {
+            Actor_MarkForDeath(&this->actor);
+        }
+    }
+}
+
+void func_80B4A168(Actor* thisx, GlobalContext* globalCtx) {
+    EnInvadepoh* this = THIS;
+
+    if ((gSaveContext.time < CLOCK_TIME(6, 0)) && (gSaveContext.time >= CLOCK_TIME(2, 0))) {
+        this->actor.update = func_80B4A1B8;
+        this->actor.draw = func_80B4E324;
+        func_80B49BD0(this);
+    }
+}
+
+void func_80B4A1B8(Actor* thisx, GlobalContext* globalCtx) {
+    s32 pad;
+    EnInvadepoh* this = THIS;
+    s32 sp2C;
+    s32 temp_v0;
+
+    sp2C = (this->actor.flags & 0x40) == 0x40;
+    temp_v0 = func_800B84D0(&this->actor, globalCtx);
+
+    if (temp_v0) {
+        func_80151BB4(globalCtx, 5);
+        func_80B49DA0(this);
+    }
+    this->actionFunc(this, globalCtx);
+    if (sp2C != 0) {
+        if (this->actor.update != NULL) {
+            SkelAnime_FrameUpdateMatrix(&this->skelAnime);
+            func_80B45CE0(&this->behaviorInfo);
+            if (this->actionFunc != func_80B49DFC) {
+                if (!temp_v0) {
+                    if (this->actor.isTargeted) {
+                        func_800B8614(&this->actor, globalCtx, 350.0f);
+                    }
+                }
+            }
+            Collider_UpdateCylinder(&this->actor, &this->collider);
+            CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
+        }
+    }
+}
+
+void func_80B4A2C0(EnInvadepoh* this) {
+    AlienBehaviorInfo* substruct = &this->behaviorInfo;
+
+    SkelAnime_ChangeAnimTransitionRepeat(&this->skelAnime, &D_06009E58, -10.0f);
+    substruct->unk26.x = 0;
+    substruct->unk26.y = 0;
+    substruct->unk26.z = 0;
+    substruct->unk30 = 0.1f;
+    substruct->unk2C = 1500;
+    this->actionTimer = Rand_S16Offset(200, 200);
+    this->unk304 = this->actor.shape.rot.y;
+    this->actor.flags |= 9;
+    this->actionFunc = func_80B4A350;
+}
+
+void func_80B4A350(EnInvadepoh* this, GlobalContext* globalCtx) {
+    s16 temp_v0;
+    s16 temp_v1_2;
+    s16 diff;
+    Vec3f sp44;
+    s16 sp42;
+    AlienBehaviorInfo* substruct = &this->behaviorInfo;
+
+    if ((globalCtx->gameplayFrames % 256) == 0) {
+        Math_Vec3s_ToVec3f(&sp44, this->pathPoints);
+        sp42 = Math_Vec3f_Yaw(&this->actor.world.pos, &sp44);
+        temp_v0 = Rand_S16Offset(-0x1F40, 0x3E80);
+        this->unk304 = temp_v0 + sp42;
+        this->behaviorInfo.unk4C = 0;
+    }
+
+    Math_StepToS(&this->behaviorInfo.unk4C, 0x7D0, 0x28);
+    Math_SmoothStepToS(&this->actor.shape.rot.y, this->unk304, 6, this->behaviorInfo.unk4C, 0x28);
+    if (this->actor.xzDistToPlayer < 300.0f) {
+        Player* player = PLAYER;
+
+        temp_v1_2 = Math_Vec3f_Pitch(&this->actor.focus.pos, &player->actor.focus.pos);
+        temp_v1_2 *= 0.85f;
+        temp_v1_2 -= this->actor.shape.rot.x;
+        substruct->unk26.x = CLAMP(temp_v1_2, -0x9C4, 0x9C4);
+        diff = (s16)(this->actor.yawTowardsPlayer - this->actor.shape.rot.y);
```

Comment:
```suggestion
        diff = BINANG_SUB(this->actor.yawTowardsPlayer, this->actor.shape.rot.y);
```
While this would fail now, this macro is being introduced in https://github.com/zeldaret/mm/pull/202/files

Reply:
#202 is merged, and you have a conflict. Please use the macro when you fix the conflict. Thanks!